src/preferences/dialog/dlgprefbeats.cpp

```diff
@@ -47,6 +47,8 @@ DlgPrefBeats::DlgPrefBeats(QWidget* parent, UserSettingsPointer pConfig)
             &QCheckBox::stateChanged,
             this,
             &DlgPrefBeats::slotReanalyzeImportedChanged);
+
+    setScrollSafeGuard(comboBoxBeatPlugin);
```

Comment:
I assume setting the scroll guard per widget is faster than using `setScrollSafeGuardForAllInputWidgets` which iterates over **all** dialog children, do an object cast etc.
To keep preferences construction (startup) fast I chose the per-widget function if there are only a few affected input widgets.
I didn't measure the impact though, maybe the difference is negligible.

For this page it probably doesn't make a big difference since there are only a few widgets.

Reply:
I don't have a strong opinion here. It can stay like this if you like.